README.md

```diff
@@ -426,6 +426,19 @@ When evaluating `function anonymous() { return this }`, we get the invocation co
 
 Getting a reference to `window` is another huge step forward for JSFuck. With the brackets characters, we could only dig in the available objects: numbers, arrays, some functions... With a reference to the global scope, we now have access to any global variable and the inner properties of these globals.
 
+### Create arbitrary function witch arguments
+
+Use following technique showed on below example to crete multi-argument function which return result: to run `[1,2,3].map( (f,t)=>f+t )` convert function inside map as follows
+
+```js
+[]["fill"]["constructor"]("return (f,t)=>f+t")()
```

Comment:
The space between `return` and `(f,t)=>f+t` is unnecessary.

Reply:
Also, if you are into a code-golf and want to reduce characters then `f,t` can be written as `["t"].concat("f")`, so the expression will become `"return"+["t"].concat("f")"+"=>f+t"`

Reply:
"showed on below" should be "showed below".

Reply:
"crete" should be "create"

Reply:
typos fixed. In example with `.map( (f,t)=>f+t )`  I want to focus on technique presentation, not on code-size optimization so I change variables to more traditional form `.map( (x,i)=>x+i )`. But @hazzik your code-size optimization suggestion are very nice